neurom/core/types.py

```diff
@@ -44,22 +44,22 @@ class NeuriteIter(OrderedEnum):
     NRN = 2
 
 
-@unique
-class NeuriteType(OrderedEnum):
-    """Enum representing valid tree types."""
-    undefined = 0
-    soma = 1
-    axon = 2
-    basal_dendrite = 3
-    apical_dendrite = 4
-    custom = 5
+# for backward compatibility with 'v1' version
+class NeuriteType(IntEnum):
+    """Type of neurite."""
+    axon = int(SectionType.axon)
+    apical_dendrite = int(SectionType.apical_dendrite)
+    basal_dendrite = int(SectionType.basal_dendrite)
+    undefined = int(SectionType.undefined)
+    soma = 31
```

Comment:
these are not int in morphio bindings ?

Reply:
I just meant : 
do you need the `int()` ?

Reply:
Ah, ok. done